lib/plugins/filter/after_render/meta_generator.js

```diff
@@ -1,13 +1,24 @@
 'use strict';
 
+const { Cache } = require('hexo-util');
+const cache = new Cache();
```

Comment:
Using local val is enough.

```js
let needInject;

...
function hexoMetaGeneratorInject(data) {
  const { config } = this;
  if (typeof needInject !== 'boolean') {
      needInject = !(!config.meta_generator
          || this.extend.helper.getProp('meta_generator')
          || data.match(/<meta([\s]+|[\s]+[^<>]+[\s]+)name=['|"]?generator['|"]?/i));
  }
```

Reply:
IMHO, use `cache.apply` here made code more readable since `cache.apply` support caching return value of given function.